mods/persistence/controllers/subsystems/trade_beacons.dm

```diff
@@ -0,0 +1,21 @@
+SUBSYSTEM_DEF(trade_beacons)
+	name = "Trade Beacons"
+	wait = 5 MINUTES
+	priority = SS_PRIORITY_MONEY_ACCOUNTS
+	var/list/all_trade_beacons = list()
+	var/list/wanted_trade_beacons = list(/obj/effect/overmap/trade_beacon/test_beacon, /obj/effect/overmap/trade_beacon/test_beacon2) // list(//obj/effect/overmap/trade_beacon/example, /obj/effect/overmap/trade_beacon/steel, /obj/effect/overmap/trade_beacon/xandahar)
+	var/last_cycle = 0
+
+/datum/controller/subsystem/trade_beacons/Initialize()
+	for(var/x in wanted_trade_beacons)
+		var/obj/effect/overmap/trade_beacon/beacon = new x()
+		beacon.Initialize()
+	. = ..()
+
+/datum/controller/subsystem/trade_beacons/fire(resumed = FALSE)
+
+	if(REALTIMEOFDAY >= (last_cycle + 2 HOURS))
```

Comment:
You could just set the subsystem's ``wait`` var to 2 HOURS instead of having to check it here and having the controller fire each 5 minutes.

Reply:
Sounds good then!